lib/livebook_web/live/hub/new_live.ex

```diff
@@ -137,30 +137,65 @@ defmodule LivebookWeb.Hub.NewLive do
               label="Livebook Teams Key"
             />
 
-            <div :if={@requested_code} class="grid grid-cols-1 gap-3">
-              <span><%= @request_code_info %></span>
-
-              <.link navigate={@verification_uri} target="_blank" class="font-bold text-blue-500">
-                <%= @verification_uri %>
-              </.link>
-
-              <span><%= @org.user_code %></span>
-            </div>
-
             <button
               :if={!@requested_code}
               class="button-base button-blue self-start"
-              phx-disable-with="Creating..."
+              phx-disable-with="Loading..."
             >
               <%= @button_label %>
             </button>
+            <div class="invisible"></div>
+            <div :if={@requested_code} class="flex flex-col rounded-xl bg-gray-50 px-10 py-6 mt-10">
+              <div class="flex flex-col items-center rounded-xl bg-gray-50">
+                <span class="text-base font-semibold text-center text-gray-900">
+                  <%= @request_code_info %>
+                </span>
+                <.link>
+                  <div class=" text-center text-sm mb-6 mt-2">
+                    <a href={@verification_uri} target="_blank" class="text-blue-600 ">
+                      Visit Livebook Teams
+                    </a>
+                    <span>
+                      and paste the code below
+                    </span>
+                  </div>
+                </.link>
+                <.copyclip content={@org.user_code} />
+              </div>
+            </div>
           </.form>
         </div>
       </div>
     </LayoutHelpers.layout>
     """
   end
 
+  defp copyclip(assigns) do
+    ~H"""
+    <div
+      id="clipboard"
+      class="flex items-center justify-between border rounded-lg px-4 py-2.5 bg-white"
+    >
+      <div class="icon-button invisible">
+        <.remix_icon icon="clipboard-line" class="text-lg" />
+      </div>
+
+      <div data-template class="text-[#e44c75] font-semibold text-xl leading-none" id="clipboard-code">
+        <%= @content %>
+      </div>
```

Comment:
We want to make sure the content has no whitespace, because the whitespace would be copied as well :)

```suggestion
      <div
        class="text-[#e44c75] font-semibold text-xl leading-none"
        id="clipboard-code"
        phx-no-format
      ><%= @content %></div>
```